petclinic/petclinic-interface/visits.smithy

```diff
@@ -65,7 +65,4 @@ structure Visit {
     /// ID of the veterinarian who saw the given pet on this visit
     @required
     vetId: U64,
-    /// The ID of the owner for this visit
-    @required     
-    ownerId: U64
-}
\ No newline at end of file
+}
```

Comment:
this would cause it to be a breaking change, but since you updated all users of this api in the same PR, The patch semver bump is ok.